### ISSUE TITLE
shinyapps deployment issue

### ISSUE THREAD

**Issue by BullzeyeBG** [rstudio/shiny#2295]
All,
I'm uploading a shiny app that works perfectly well on my local machine.
I had a big trouble getting it work (received the error on the picture below) on shinyapps.io but it happened finally, not sure how.

Now that it worked it worked only once. If I upload another application and try running the first I get the same error from the picture below. Then I get the same error for the secondly uploaded app (they are different applications).
Only when I "Restart Application" it works for once. This does not seem nice if it happens to users that you share the app with.
ANy ideas?
<img width="953" alt="capture" src="https://user-images.githubusercontent.com/11310531/50662439-13f49400-0faf-11e9-80ed-65895e7cbcd0.PNG">

**Comment by ajmcoqui** [comment]
Good morning! This repository is for issues with the `shiny` package itself, but it looks like you're having an issue with the shinyapps.io service. If you have a paid account (Starter, Basic, Standard, or Pro), please file a support ticket via https://support.rstudio.com, and if you have a Free account, please post to the RStudio Community with the `shinyappsio` tag: https://community.rstudio.com/tags/shinyappsio. Please include a small code snippet that reproduces the error, to give the community members an idea of what you're trying to do.

That said, this particular error appears to be in your code, rather than in the service. I did a quick search and found the following references online:

[RStudio Community post](https://community.rstudio.com/t/number-of-items-to-replace-is-not-a-multiple-of-replacement-length/17059/4)
[Another RStudio Community post](https://community.rstudio.com/t/warning-error-in-data-frame-replacement-has-2-rows-data-has-1-shiny/12801/2)
[StackOverflow post](https://stackoverflow.com/questions/41157681/r-shiny-error-number-of-items-to-replace-is-not-a-multiple-of-replacement-lengt)

Your code may work locally because it is picking up objects from your global environment that are not available on a clean application instance in shinyapps.io.

**Comment by alandipert** [comment]
Closing for the reasons mentioned by @ajmcoqui, but if you continue to have problems please don't hesitate to comment here further or open a new issue.
